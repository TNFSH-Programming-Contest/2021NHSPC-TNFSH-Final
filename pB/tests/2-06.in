8 9 7
4 2 3 1 6 7
9 8 3 2 6 7
6 8 4 9 3 1
2 1 4 6 5 3
2 8 5 9 3 6
5 1 3 2 4 8
8 7 9 4 6 3
2 5 9 8 4 3
4 1 3 4 6
6 1 8 6 2 4 5
8 6 3 8 5 7 4 1 2
6 4 3 8 7 1 6
4 5 4 6 8
6 3 5 7 2 4 1
3 1 7 2
6 8 7 6 3 5 2
5 5 8 7 2 3
