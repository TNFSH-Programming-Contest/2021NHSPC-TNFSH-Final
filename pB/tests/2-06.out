0
3 4 5 8
0
1 1
1 6
1 3
0
1 7
1 2
